Wrappers/Python/ccpi/optimisation/functions/KullbackLeibler.py

```diff
@@ -54,7 +57,7 @@ def __call__(self, x):
         '''Evaluates KullbackLeibler at x'''
 
         ind = x.as_array()>0
-        tmp = scipy.special.kl_div(self.b.as_array()[ind], x.as_array()[ind])                
+        tmp = scipy.special.kl_div(self.data.as_array()[ind], x.as_array()[ind])                
```

Comment:
seems that this forgot about `background_noise`. Note that if it needs to be there, then the `ind` calculation above using `x` is probably not what you want as well. (isn't that condition surprising by the way. should at least be documented)

Reply:
You're right, I will fix this. For the ind, i consider positive values so not to have NaN values when computing KL from [scipy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.kl_div.html)